rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantCreator.java

```diff
@@ -1,5 +1,9 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
 
+import static pl.oaza.warszawa.dor.rekolekcje.api.participants.domain.DateConverter.convertToDateTime;
+
+import java.util.Set;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'java\.util\.Set' is in the wrong order\. Should be in the 'STANDARD\_JAVA\_PACKAGE' group, expecting group 'THIRD\_PARTY\_PACKAGE' on this line\.